.gitattributes

```diff
@@ -4,3 +4,7 @@
 Makefile text eol=lf
 configure text eol=lf
 .gitmodules text eol=lf
+# The following files are generated and should not be shown in the
+# diffs by default on Github.
+doc/lightning*.7 linguist-generated=true
```

Comment:
Agree with this in principle but this was helpful once when my generator code was out of date and @rustyrussell noticed.

Reply:
Good point, we might want to hold off on this one.

Reply:
What's the chance it'll happen twice?  :)

I'm OK with it, TBH.  Should probably include doc/lightning-cli.1 though.